doc/rls_mode.md

```diff
@@ -14,39 +14,92 @@ The type of the parameter is an object with the following fields:
 
 By default, it is `null`.
 
-### Debugging
+## Setting up
 
-There is an output channel named "Rust Language Server" which is used to show messages from RLS.
+First of all, you have to download the RLS sourcs:
 
-To open it, perform the following steps:
+```bash
+git clone https://github.com/rust-lang-nursery/rls
+```
 
-* Click "View" on the menu
-* Click "Output" on submenu
-* Click on the listbox which is to the right of the shown panel
-* Choose "Rust Language Server"
+Depending on whether you have rustup or not, there are different ways you can set up this plugin.
+
+* [Setting up without rustup](#without-rustup)
+* [Setting up with rustup](#with-rustup)
+
+#### Without rustup
```

Comment:
Leaving these instructions in place might be useful to some people. The RLS repository recommended avoiding `rustup` as it has caused problems in the past, though this may be outdated information now.

If `rustup` is the recommended method now, I think that it should be pointed out. Also, the "Setting up with rustup" section should then be listed first.